File: app/src/main/java/com/apps/adrcotfas/goodtime/Main/ItemTouchHelperViewHolder.java
package com.apps.adrcotfas.goodtime.Main;

import androidx.recyclerview.widget.ItemTouchHelper;

public interface ItemTouchHelperViewHolder {

    /**
     * Called when the {@link ItemTouchHelper} first registers an item as being moved or swiped.
     * Implementations should update the item view to indicate it's active state.
     */
    void onItemSelected();
}

File: app/src/main/java/com/apps/adrcotfas/goodtime/main/ItemTouchHelperViewHolder.kt
package com.apps.adrcotfas.goodtime.main

interface ItemTouchHelperViewHolder {
    /**
     * Called when the [ItemTouchHelper] first registers an item as being moved or swiped.
     * Implementations should update the item view to indicate it's active state.
     */
    fun onItemSelected()

    /**
     * Called when the [ItemTouchHelper] has completed the
     * move or swipe, and the active item state should be cleared.
     */
    fun onItemClear()
}